tests/resources/parser/test_data/us/APNews.json

```diff
@@ -25,64 +25,129 @@
+            "“Post Labor Day, when the bell rings, there is a battle for a slim universe of — you can call them anything you want: persuasion voters, swing voters, independent voters — and it’s pretty small, and that’s where each side gets a billion dollars,” Anzalone said.",
+            "Many independents appear to find both candidates unsatisfying, according to an AP-NORC poll conducted in August.",
+            "For now, Harris also has a slight advantage on some key traits among independents, while she and Trump are about even on others.",
+            "For example, about 3 in 10 independents say that “honest” describes Harris better, while about 2 in 10 say it describes Trump better. About 3 in 10 also say that “committed to democracy” describes Harris better, while less than 2 in 10 say it describes Trump better.",
+            "The candidates were about equally likely to be perceived by independents as capable of winning the election, capable of handling a crisis, and “caring about people like you.”"
+          ]
+        },
+        {
+          "headline": [
+            "Who is the ‘change candidate’?"
+          ],
+          "paragraphs": [
+            "The race may ultimately be decided by whichever candidate can most successfully cast themselves as the “change candidate” given that about 7 in 10 voters say the country is heading in the wrong direction, based on an AP-NORC poll conducted in late July after Biden withdrew from the race.",
+            "Trump was the face of change when he won the 2016 election. And even after serving in the White House for four years, he continues to energize millions of frustrated voters who embrace his brash leadership style and unwillingness to follow the traditional rules of politics.",
+            "Harris has been Biden’s vice president for nearly four years, yet the historic nature of her candidacy — she would be the first woman president — allows her to make a convincing case that she represents a new direction for the country, said veteran Democratic strategist James Carville.",
+            "Still, he’s worried about his party’s “severe underperformance” in the so-called “blue wall” states in recent elections.",
+            "“I’ll feel good after the election,” Carville said. “Let’s get the hay in the barn. There’s still a lot of hay out there in the field.”"
           ]
         }
       ]
     },
-    "publishing_date": "2023-07-11 15:19:21+00:00",
-    "title": "Microsoft can move ahead with record $69 billion acquisition of Activision Blizzard, judge rules",
+    "publishing_date": "2024-09-03 04:03:19+00:00",
+    "title": "The presidential campaigns brace for a sprint to Election Day",
     "topics": [
-      "Microsoft Corp",
-      "Gaming",
-      "Activision Blizzard Inc",
+      "Kamala Harris",
+      "Donald Trump",
+      "Pennsylvania",
+      "Georgia",
       "General news",
+      "Domestic News",
+      "NE State Wire",
+      "MN State Wire",
+      "ME State Wire",
```

Comment:
This seems to be a whole lot of topics. I would consider the filtering the following three types:
- Anything containing `State Wire`
- One letter topics such as: p, s, w, n etc
- Anything containing news
I think this should still leave us with enough meaningful topics.